xdocs/usermanual/component_reference.xml

```diff
@@ -6070,7 +6073,6 @@ The JDBC PostProcessor enables you to run some SQL statement just after a sample
 This can be useful if your JDBC Sample changes some data and you want to reset state to what it was before the JDBC sample run.
 </p>
 </description>
-</component>
```

Comment:
This change of placing the `</component>` should be done separately, since it is a bug in the current doc.